### PR TITLE
 Scripts on Wizard

### PR DESCRIPTION
**Pull request by Williangalvani**
Needs more testing, but looking good so far. I'll review it tomorrow

### CODE REVIEW

**Comment by Williangalvani** [comment]
depends on https://github.com/bluerobotics/Blueos-Parameter-Repository/pull/6

File: core/frontend/src/components/wizard/ScriptLoader.vue [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+import { SemVer } from 'semver'
+import Vue from 'vue'
+
+import autopilot from '@/store/autopilot_manager'
+import { Firmware, Vehicle } from '@/types/autopilot'
+import { callPeriodically, stopCallingPeriodically } from '@/utils/helper_functions'
+
+import { availableFirmwares, fetchCurrentBoard } from '../autopilot/AutopilotManagerUpdater'
+
+const REPOSITORY_ROOT = 'https://williangalvani.github.io/Blueos-Parameter-Repository_/'
```
Comment (Williangalvani) [inline core/frontend/src/components/wizard/ScriptLoader.vue:49]: update after https://github.com/bluerobotics/Blueos-Parameter-Repository/pull/6

File: core/frontend/src/components/wizard/Wizard.vue [hunk 2]
```diff
@@ -284,9 +289,12 @@ import { sleep } from '@/utils/helper_functions'
 import ActionStepper, { Configuration, ConfigurationStatus } from './ActionStepper.vue'
 import DefaultParamLoader from './DefaultParamLoader.vue'
 import RequireInternet from './RequireInternet.vue'
+import ScriptLoader from './ScriptLoader.vue'
 
 const WIZARD_VERSION = 4
 
+const REPOSITORY_ROOT = 'https://williangalvani.github.io/Blueos-Parameter-Repository_/'
```
Comment (Williangalvani) [inline core/frontend/src/components/wizard/Wizard.vue:296]: update after https://github.com/bluerobotics/Blueos-Parameter-Repository/pull/6

**Review by patrickelectric — CHANGES_REQUESTED** [review]

File: core/frontend/src/components/wizard/Wizard.vue [hunk 3]
```diff
@@ -648,6 +667,34 @@ export default Vue.extend({
         })
         .catch((error) => `Failed to fetch available firmware: ${error.message ?? error.response?.data}.`)
     },
+    async installScripts(): Promise<ConfigurationStatus> {
+      const scripts_folder = 'configs/ardupilot-manager/firmware/scripts/'
+      await filebrowser.createFolder('')
```
[the remark this answers is not in the document]
Comment (Williangalvani) [reply core/frontend/src/components/wizard/Wizard.vue:672]: yep, this is wrong

[the remark this answers is not in the document]
Comment (Williangalvani) [reply core/frontend/src/components/wizard/Wizard.vue:672]: the upside is we found out we don't need to explicitly create the folder

**Review by patrickelectric — APPROVED** [review]

**Comment by patrickelectric** [comment]
@Williangalvani can you cherry-pick to 1.2 ?

**Pull request merged**